base/sim/FairDetector.h

```diff
@@ -99,6 +99,8 @@ class FairDetector : public FairModule
       return fDetId;
     }
 
+    virtual void DefineSensitiveVolumes();
```

Comment:
I am wondering if this is this a new **public** interface of FairDetector? Are clients supposed to implement it?
(To me it looks like there is a default implementation in FairDetector and possibly a non-virtual private method would be sufficient here)

Reply:
This part has been implemented by Ivana to make the MT running
and removed some time ago, also by Ivana.
It seems however, that without this part the G4MT is not working.

Reply:
Sure. There is no question if this part is necessary (it seems to be called from somewhere). I am just wondering if the scope of the function can be reduced to non-virtual private as otherwise users might start to override and thereby breaking the simulation.

Reply:
That makes sense. Thanks.